source-mysql/main.go

```diff
@@ -314,6 +319,38 @@ func (db *mysqlDatabase) connect(ctx context.Context) error {
 	return nil
 }
 
+func (db *mysqlDatabase) logServerVariables(level logrus.Level) error {
+	if !logrus.IsLevelEnabled(level) {
```

Comment:
Having the log level as an argument here was a little confusing since we only ever call it with the debug level from line 271 and I was looking for other places it was called with different log levels and couldn't find any. Maybe we could hard-code the check here to return early on `!logrus.IsLevelEnabled(logrus.DebugLevel)` and make the logging on line 350 always at the `debug` level?

Reply:
Hmm, good point. Previously I had the "is it enabled" check wrapped around the call to `logServerVariables` over in the connect function, and so the level was a parameter to avoid stating it in two separate places and getting them out of sync. But yeah, now that this check is also part of the function it's unnecessary.